### PR TITLE
feature/course-2

### PR DESCRIPTION
**Pull request by RoseKoning**

### CODE REVIEW

File: staff/rose-koning/project-2/sprint-4/task-6/congress/home.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+          <a class="nav-link disabled" href="#">Contact</a>
+        </li>
+      </ul>
+    </nav>
+    <section class="container-fluid">
+        <h1>Welcome</h1>
+        <div class="row">
+            <div class="col">
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus imperdiet, nulla et dictum interdum, nisi lorem egestas vitae scel<span id="dots">...</span><span id="more">erisque enim ligula venenatis dolor. Maecenas nisl est, ultrices nec congue eget, auctor vitae massa. Fusce luctus vestibulum augue ut aliquet. Nunc sagittis dictum nisi, sed ullamcorper ipsum dignissim ac. In at libero sed nunc venenatis imperdiet sed ornare turpis. Donec vitae dui eget tellus gravida venenatis. Integer fringilla congue eros non fermentum. Sed dapibus pulvinar nibh tempor porta. Lorem ipsum dolor sit amet consectetur adipisicing elit. Quibusdam, reprehenderit perferendis sunt ut eaque, dolore cupiditate quos similique voluptate totam culpa. Deleniti itaque officia similique cum esse. Nemo, debitis deserunt.</span></p>
+                <button onclick="myFunction()" id="myBtn">Read more</button>
```
Comment (manuelbarzi) [inline staff/rose-koning/project-2/sprint-4/task-6/congress/home.html:87]: correct the behavior error when clicking on this button => 

```js
home.html:87 Uncaught ReferenceError: myFunction is not defined
    at HTMLButtonElement.onclick (home.html:87)
```

File: staff/rose-koning/project-2/sprint-3/task-5/legislators.html [hunk 2]
```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
```
Comment (manuelbarzi) [inline staff/rose-koning/project-2/sprint-3/task-5/legislators.html:1]: correct the links to Home, Congress 113, Attendance. right now they do not work => `Cannot GET /staff/rose-koning/project-2/sprint-3/task-5/ ...`

File: staff/rose-koning/project-2/sprint-4/task-6/congress/home.html (hunk 1 above)
Comment (RoseKoning) [reply staff/rose-koning/project-2/sprint-4/task-6/congress/home.html:87]: This function is working on my computer. The function myFunction is located in the main

File: staff/rose-koning/project-2/sprint-3/task-5/legislators.html (hunk 2 above)
Comment (RoseKoning) [reply staff/rose-koning/project-2/sprint-3/task-5/legislators.html:1]: fixed

File: staff/rose-koning/project-2/sprint-4/task-6/congress/house-data.html [hunk 3]
```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
```
Comment (manuelbarzi) [inline staff/rose-koning/project-2/sprint-4/task-6/congress/house-data.html:1]: house data fails on page load
```js
congress-data.js:1 Uncaught ReferenceError: url is not defined
    at congress-data.js:1
(anonymous) @ congress-data.js:1
```

File: staff/rose-koning/project-2/sprint-4/task-6/congress/home.html [hunk 4]
```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
```
Comment (manuelbarzi) [inline staff/rose-koning/project-2/sprint-4/task-6/congress/home.html:1]: congress and attendance links point to the same pages, but they should show different pages with different information. attendance pages are missing here in this task-6. please, add them and correct the links.

**Review by manuelbarzi — CHANGES_REQUESTED** [review]
review the comments and update the project to make it work property.

**Pull request merged**